packages/matchbox/src/components/ComboBox/ComboBoxTextField.js

```diff
@@ -101,14 +105,18 @@ function ComboBoxTextField(props) {
           onKeyDown={handleKeyDown}
           placeholder={placeholder}
           readOnly={readOnly}
-          ref={inputRef}
+          required={required}
           style={style}
+          type="text"
           value={value}
+          {...inputProps}
         />
-      </div>
-      {error && !errorInLabel && <Error error={error} />}
-      {helpMarkup}
-    </fieldset>
+      </StyledInputWrapper>
+      {/* Menu is rendered here so it is positioned correctly before error and helptext */}
+      {renderMenu && renderMenu}
```

Comment:
Hm not sure what you mean. I think `React.Children` needs to be called from the context of the wrapper since the menu will be a sibling of this component. Or did you mean pass it with `props.children` instead of `props.renderMenu`?